HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Resources/de.lproj/Localizable.strings

```diff
@@ -53,3 +53,9 @@
 "ginihealth.paymentcomponent.installAppBottom.notes.description" = "Hinweis: Bitte aktualisieren oder installieren Sie zunächst die [BANK]-App aus dem App-Store.";
```

Comment:
[Suggestion]: @razvancapra  could you please add `sheet` to the keys `ginihealth.paymentcomponent.installAppBottom` -> `ginihealth.paymentcomponent.installAppBottomSheet`